csm_web/scheduler/urls.py

```diff
@@ -23,4 +23,7 @@
     path("matcher/<int:pk>/mentors/", views.matcher.mentors),
     path("matcher/<int:pk>/configure/", views.matcher.configure),
     path("matcher/<int:pk>/create/", views.matcher.create),
+    path(
+        "search/get-sections/", views.get_sections_of_user, name="get_sections_of_user"
```

Comment:
Generally, the path itself shouldn't need to include actions like "get"; the HTTP method that the request has should indicate what the user wants. The more conventional path would be something like `search/sections/`.